app/mailers/patient_mailer.rb

```diff
@@ -168,6 +169,7 @@ def assessment_email(patient)
     return unless patient.last_assessment_reminder_sent_eligible?
 
     @lang = patient.select_language
+    @contact_info = Jurisdiction.find_by_id(patient.jurisdiction_id).contact_info
```

Comment:
Same as above

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/925/commits/860024d23a6f4b689c1baa9bc9cd315a5e456796